### ISSUE TITLE
Recent Payouts chart broken

### ISSUE THREAD

**Issue by Gabilin** [paritytech/polkadot-staking-dashboard#541]
On Jan 1, 2023, the Recent Payouts chart started showing wrong data:

- instead of the last 19 days (as per code), the timeline is for 6 month 
- on Jun 20th, teh chart shows unnormous 1467 DOT reward - this never happaned
- 5 moth skipped after the Jun 20th. The rest of the rewirds are squized due to scaling.

I have tryed cleaning brother cach and recoonect the Polkadot JS
Reference image:
![image](https://user-images.githubusercontent.com/10139216/211071738-d57b6361-d33c-416f-8a43-77d80e8f2410.png)

**Comment by rossbulat** [comment]
This has been fixed in a recent update & we are waiting on it to be merged into production. Github deployment contains the latest fixes - follow me on Twitter to get updates on fixes.
